libclamav_rust/src/fuzzy_hash.rs

```diff
@@ -68,6 +68,9 @@ pub enum FuzzyHashError {
 
     #[error("{0} parmeter is NULL")]
     NullParam(&'static str),
+
+    #[error("{0} hash must be 16 characters in length")]
+    InvalidHashLength(&'static str),
```

Comment:
Are all hashes going to be 16 bytes?

Also, did you mean bytes? `char` is a different beast outside of C.

```suggestion
    #[error("{0} hash must be {1} bytes in length")]
    InvalidHashLength(&'static str, usize),
```

Reply:
That message is a copy-paste but yeah I may as well improve while I'm touching it. bytes is better, and customizing the hash length is better.

Reply:
Actually, characters makes sense in this context because it's talking about a hex string -> bytes conversion.  Each char of a hex string is just a nibble.

Reply:
ah...